security/custom_authenticator.rst

```diff
@@ -255,6 +262,89 @@ using :ref:`the user provider <security-user-providers>`::
             }
         }
 
+It is a good practice to normalize the user identifier before using it.
+For example, this ensures that variations such as "john.doe", "John.Doe",
+or "JOHN.DOE" refer to the same user.
+Normalization can include converting the identifier to lowercase
+and trimming unnecessary spaces.
+You can optionally pass a user identifier normalizer as third argument to the
+``UserBadge``. This callable receives the ``$userIdentifier``
+and must return a normalized user identifier as a string.
+
+.. versionadded:: 7.3
```

Comment:
New section is not correctly rendered:

![image](https://github.com/user-attachments/assets/d5560800-f910-441b-a142-c98e4f30e1bb)

I don't know how to fix it

Reply:
Many thanks.
I will take time to update this section.

Reply:
It should be fixed with the last commit